model/src/main/proto/feedback_reporting.proto

```diff
@@ -324,8 +325,11 @@ message FeedbackReportingAppContext {
   }
   // The text size the app is set to.
   ReadingTextSize text_size = 5;
+
+  // Reserve 6 which was used before as using it might cause import issues for older profiles.
+  reserved 6;
   // The text language the app is set to.
-  AppLanguage text_language = 6;
+  OppiaLanguage app_language = 12;
```

Comment:
This should be index 10 since 10 and 11 haven't been used yet.

Reply:
updated.